packages/protocol/contracts/ACE/validators/bilateralSwap/BilateralSwapABIEncoder.sol

```diff
@@ -34,14 +34,14 @@ library BilateralSwapABIEncoder {
             // memory map of `proofOutputs`
             // 0x00 - 0x160  = scratch data for EIP712 signature computation and note hash computation
             // ACE_NOTE_SIGNATURE struct hash variables
-            // 0x80 = struct hash
-            // 0xa0 = proofId (1)
+            // 0x80 = type hash
+            // 0xa0 = proof object (65537)
```

Comment:
this should have a different proof ID to JoinSplit. The IDs of our initial proofs are:

* JoinSplit: 1  
* BilateralSwap: 2  
* DividendComputation: 3

Reply:
wait, on second thoughts this contract shouldn't be validating ECDSA signatures. This might require changes in a different branch

Reply:
@thomas-waite told me that those values are residual and they were mistakenly kept in the abi encoder when he took the template from join split. If that's not true, I guess the value of the proof object should be 65538?

Reply:
Fixed in https://github.com/AztecProtocol/AZTEC/pull/75